core/src/main/java/org/wildfly/extras/creaper/core/online/ManagementProtocol.java

```diff
@@ -10,11 +10,15 @@
     /**
      * HTTP transport is used. Default port 9990
      * Asynchronous execution is not provided.
-     * If an error occurs (server does not respond 401 or header does not contain WWW-Authenticate field after
-     * first request) {@code IllegalStateException is thrown}
      * Execution on is allowed with <b>no attachments</b> (i.e. deployment operation).
      */
     HTTP("http"),
+    /**
+     * HTTPS transport is used. Default port 9993
+     * Asynchronous execution is not provided.
+     * Execution on is allowed with <b>no attachments</b> (i.e. deployment operation).
+     */
+    HTTPS("https"),
```

Comment:
Hmm, OK, but then `HttpModelControllerClient.https` should check if `SslOptions` are not `null`, no?